matlab/coordinate_transforms/R2AA.m

```diff
@@ -0,0 +1,16 @@
+function AA = R2AA(R)
```

Comment:
True, I tried it without normalization, but the more thorough method would be to explicitly normalize (and check for large enough rotations).